File: Examples/VisualBasic/AsposePDF/Pages/GetParticularPage.vb
Imports System.IO
Imports System
Imports Microsoft.VisualBasic
Imports Aspose.Pdf
Namespace AsposePDF.Pages
    Public Class GetParticularPage
        Public Shared Sub Run()
            ' ExStart:GetParticularPage
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_AsposePdf_Pages()

            ' Open document
            Dim pdfDocument As New Document(dataDir & Convert.ToString("GetParticularPage.pdf"))

            ' Get particular page
            Dim pdfPage As Page = pdfDocument.Pages(2)

            ' Save the page as PDF file
            Dim newDocument As New Document()

            newDocument.Pages.Add(pdfPage)

            dataDir = dataDir & Convert.ToString("GetParticularPage_out_.pdf")
            newDocument.Save(dataDir)
            ' ExEnd:GetParticularPage
            System.Console.WriteLine(Convert.ToString(vbLf & "Particular page accessed successfully." & vbLf & "File saved at ") & dataDir)
        End Sub
    End Class
End Namespace

File: Examples/CSharp/AsposePDF/Pages/GetParticularPage.cs
//////////////////////////////////////////////////////////////////////////
// Copyright 2001-2013 Aspose Pty Ltd. All Rights Reserved.
//
// This file is part of Aspose.Pdf. The source code in this file
// is only intended as a supplement to the documentation, and is provided
// "as is", without warranty of any kind, either expressed or implied.
//////////////////////////////////////////////////////////////////////////
using System.IO;

using Aspose.Pdf;

namespace CSharp.AsposePdf.Pages
{
    public class GetParticularPage
    {
        public static void Run()
        {
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_AsposePdf_Pages();

            //open document
            Document pdfDocument = new Document(dataDir + "GetParticularPage.pdf");

            //get particular page
            Page pdfPage = pdfDocument.Pages[2];

            //save the page as PDF file
            Document newDocument = new Document();

            newDocument.Pages.Add(pdfPage);

            newDocument.Save(dataDir + "GetParticularPage_out.pdf");

        }
    }
}